packages/dai/test/eth/EthereumCdpService.spec.js

```diff
@@ -50,6 +51,36 @@ describe('find cdp', () => {
     expect(sameCdp.dsProxyAddress).not.toBeDefined();
   });
 
+  test('regression: handle null proxy correctly', async () => {
+    const cdps = buildTestEthereumCdpService({
+      accounts: {
+        default: { type: 'privateKey', ...TestAccountProvider.nextAccount() }
+      }
+    });
+    await cdps.manager().authenticate();
+    const saved = { get: cdps.get };
```

Comment:
you don't need the stubs anymore :)

Reply:
I missed the full scope of what `TestAccountProvider` allowed me to do.  I've been trying to code this while in meetings all day.  Works perfectly and is much cleaner now.